source/2017-04-13-how-we-got-here.html.markdown.erb

```diff
@@ -0,0 +1,147 @@
+it didn't have any cabling at all. Since then all visible and non-visible LAN
+outlets were patched and if you spot an outlet at our Colab feel free to use it.
+
+![Before and after](beforeAfter.jpg)
+
+With the help of our Network crack (koa) we've got a solid setup and started to
+use Mikrotiks Capsman, which is a bit of a pain to get into but it works like a
+charm after the initial Setup (and 10 Factory Resets later) and a little bit of
+a hassle to wrap your head around it.
+
+## Perception
+There's always a gap between sysadmins and users when it comes to quality, we're
+pretty much the only fans of cabled connection and we love the convenience of cables
+and because we had a lot of LAN Ports in this building where a school was located
+back in the day, we had this sweet vision of small AP & Switch combos on every
+table so people who preferred WiFi could get it from the ether and the other
+half could use cables as they desire, but well, that idea got chased out of town
+pretty fast (Except the [makerZspace](https://makerzspace.ch/) they love cables and got them).
+
+So we decided to aim for total 5g coverage and slowly starve out 2.4g, because it's
+way faster and it allows you to put up a higher density of AP's and if less people use
+one access point everyone will have a faster connection, and since we where living in
+the year 2015 where every device is capable of 5g there shouldn't be an issue with
+partially dropping 2.4g, right? **TOTALLY WRONG**
+
+![ClientBandDistribution](ClientBandDistribution.jpg)
+
+As it turns out the 5g capable devices aren't consistently great with 5g connectivity,
+most of those devices only support the lower bands 5-036 & 5-044 and hollow glass
+bricks are apparently the death of every 5g connection, there are also quite some
+differences between mikrotiks different products, here's a list of a few of which
+we went through and the ones we're currently using.
+
+### Legacy AP's
+* [Netmetal](https://routerboard.com/RB922UAGS-5HPacT-NM) - Alot of DIY, so it's
+time consuming and thus not quite cheap, but nearly indestructible and perfect
+for outdoors
+* [Netbox](https://routerboard.com/RB911G-5HPacD-NB) - Also DIY, pretty huge and
+not quite cheap, also not the most aesthetic ones
+* [cAP2n](https://routerboard.com/RBcAP2nD) - Plastic toy, wouldn't recommend
+
+### Current AP's
+* [wAP](https://routerboard.com/RBwAPG-5HacT2HnD) - Aesthetic pretty good
+dualband accesspoints with a good range
+* [SXT SA5 ac](https://routerboard.com/RBSXTG-5HPacD-SA) - With an angle of 90°
+perfect for the corners in our CoWorking Space
+* [wAP ac](https://routerboard.com/RBwAPG-5HacT2HnD) - very fast, performant AP
+* [hAP ac](https://routerboard.com/RB962UiGS-5HacT2HnT) - like the wAP ac but with
+a built in switch
+
+We were also pretty harsh when it comes down to kicking people out of our Accesspoints,
+because if you got a bad connection you're giving a bad time to all the other people
```

Comment:
got as in 'if you have gotten'
Du bekommst nicht eine schlechte verbindung, du hast zu diesem Zeitpunkt eine schlechte verbindung

> Get is the present tense form of the verb. Got is the past tense form as well as one of the two alternatives for the past participle. The other alternative for the past participle is gotten, which is generally preferred in the United States.